package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mappet",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Comment:
This commit is not a version bump. See:
https://github.com/MichalZalecki/mappet/blob/fb27134436bb1ac227656852ef1a85081c5a1271/package.json#L3
It's just what `npm install` does.